### PR TITLE
TELCODOCS-1338 - adding RAN 4.13 post-GA known issues

### PR DESCRIPTION
**Pull request by aireilly**
Adds RAN 4.13 post GA known issues. 

Version(s):
enterprise-4.13 only

Issue:
https://issues.redhat.com/browse/TELCODOCS-1338

Link to docs preview:
https://60566--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-ran-known-issues-post-ga

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:2788]: @yliu127 Can you review/lgtm on this known issue text for https://issues.redhat.com/browse/OCPBUGS-8434 please.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:2801]: @trwest@redhat.com PTAL I need a QE lgtm on this known issue text please.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:2803]: @chadcrum Can you review/lgtm on this known issue text for https://issues.redhat.com/browse/OCPBUGS-4388 please.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:2808]: @reihl Can you review/lgtm on this known issue text for https://issues.redhat.com/browse/OCPBUGS-13138 please.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (trewest) [reply release_notes/ocp-4-13-release-notes.adoc:2801]: /lgtm

**Comment by tradej** [comment]
LGTM for docs (Change Management)

File: release_notes/ocp-4-13-release-notes.adoc
Comment (kcarmichael08) [inline release_notes/ocp-4-13-release-notes.adoc:2800]:
```suggestion
* After installing a dual-stack {ztp} hub cluster, enabling dual-stack Virtual IP addresses (VIPs), and enabling the `virtualMediaViaExternalNetwork` flag in a `Provisioning` CR, the `IRONIC_EXTERNAL_URL_V6` environment variable incorrectly gets assigned an IPv4 address.
```

File: release_notes/ocp-4-13-release-notes.adoc
Comment (kcarmichael08) [inline release_notes/ocp-4-13-release-notes.adoc:2805]:
```suggestion
The `FirmwareSchema` CR generated for the ZT server doesn't have the `allowable_values`, `attribute_type`, and `read_only` fields populated.
```

**Review by kcarmichael08 — COMMENTED** [review]
Couple of suggestions; otherwise lgtm!

File: release_notes/ocp-4-13-release-notes.adoc
Comment (yliu127) [reply release_notes/ocp-4-13-release-notes.adoc:2788]: /lgtm

File: release_notes/ocp-4-13-release-notes.adoc
Comment (reihl) [reply release_notes/ocp-4-13-release-notes.adoc:2808]: correct, but worth mention that it's related to HW (HP 10Gen+ in my case, could be in other platforms as well)

Comment (reihl) [reply release_notes/ocp-4-13-release-notes.adoc:2808]: re-test in on 4.13.0 the problem is there as well. we don't need to specify RC.x just 4.13.0

**Review by reihl — APPROVED** [review]
/LGTM

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:2800]: LGTM from @trewest Via Slack 26/05/2023

Comment (trewest) [reply release_notes/ocp-4-13-release-notes.adoc:2800]: /lgtm

File: release_notes/ocp-4-13-release-notes.adoc
Comment (kelbrown20) [inline release_notes/ocp-4-13-release-notes.adoc:2803]: I'm pretty sure this needs a bullet point here. 
```suggestion
* ZT servers have the `BiosRegistry` language set to `en-US` instead of `en`.
```

**Review by kelbrown20 — COMMENTED** [review]
Hi @aireilly, great work here! Just a small nit before merging.

**Comment by aireilly** [comment]
> Hi @aireilly, great work here! Just a small nit before merging.

Updated. Merge at your convenience please :)

**Review by kelbrown20 — COMMENTED** [review]
LGTM! Thank you for for updating!

**Pull request merged**
